src/components/inputs/react-query-builder/combinator-selector.tsx

```diff
@@ -7,7 +7,7 @@
 
 import { CombinatorSelectorProps } from 'react-querybuilder';
 import { FunctionComponent, useCallback, useState } from 'react';
-import PopupConfirmationDialog from '../dialogs/popup-confirmation-dialog';
+import PopupConfirmationDialog from '../../dialogs/popup-confirmation-dialog.tsx';
```

Comment:
remove extension ?

Reply:
this said it seems there's no clear rule on this matter for imports

Reply:
Yes it's my IDE which add it, thanks.
It's better without.